pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -33,24 +32,41 @@ const (
 )
 
 // Returns a URI to the explainer. This URI is passed to the model-initializer via the ModelInitializerSourceUriInternalAnnotationKey
-func (m *ExplainerSpec) GetStorageUri() string {
-	return getExplainerHandler(m).GetStorageUri()
+func (e *ExplainerSpec) GetStorageUri() string {
+	explainer, err := getExplainer(e)
+	if err != nil {
```

Comment:
Seems strange to swallow the error here?

Reply:
Yeah, you're right, it's strange.
I didn't return error because I didn't want to introduce different approach from what we already have on predictor and transformer. If we want to return error here, I think it's better to update predictor and transformer as well, so we have a uniform approach.
We will have more changes though but I believe it will lead us to better code. What do you think?

Reply:
I would refactor all unless there is some good reason to swallow errors.

Reply:
@ellis-bigelow any thoughts?

Reply:
We need to swallow these errors as mentioned elsewhere.